app/code/Magento/CmsGraphQl/Model/Resolver/DataProvider/Block.php

```diff
@@ -48,7 +48,7 @@ public function getData(string $blockIdentifier): array
     {
         $block = $this->blockRepository->getById($blockIdentifier);
 
-        if (false === $block->isActive()) {
+        if (!$block->getId()) {
```

Comment:
Hi, Ronak
We can completely delete this check because we already have this check in `\Magento\Cms\Model\BlockRepository::getById`
```
public function getById($blockId)
    {
        $block = $this->blockFactory->create();
        $this->resource->load($block, $blockId);
        if (!$block->getId()) {
            throw new NoSuchEntityException(__('The CMS block with the "%1" ID doesn\'t exist.', $blockId));
        }
        return $block;
    }
```
Also, need to update API-functional tests according to changes in functionallity
If uou need some help or examples, pls, ping me in Slack channel
Thanks